### PR TITLE
Add pycharm professional 2018.3.3

### PR DESCRIPTION
**Pull request by edgardmessias**

### CODE REVIEW

File: bucket/pycharm-pro.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
+    "version": "2018.3.3",
+    "homepage": "https://www.jetbrains.com/pycharm/",
+    "url": "https://download.jetbrains.com/python/pycharm-professional-2018.3.3.zip",
+    "hash": "cd9da42dcf3bf5066957df4d521a63d6b314da055ac0c382d51deeb08762eef6",
+    "env_add_path": "bin",
```
Comment (Ash258) [inline bucket/pycharm-pro.json]: Why this add path?? In bin folder are only internal application to help restarting, notifying, other small operations.

File: bucket/pycharm-pro.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            "bin": [
+                "bin\\pycharm64.exe",
+                [
+                    "bin\\pycharm64.exe",
+                    "pycharm"
+                ]
+            ],
+            "shortcuts": [
+                [
+                    "bin/pycharm64.exe",
```
Comment (Ash258) [inline bucket/pycharm-pro.json]: Inconsistent slashes in shortcuts.

File: bucket/pycharm-pro.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
+{
+    "version": "2018.3.3",
+    "homepage": "https://www.jetbrains.com/pycharm/",
+    "url": "https://download.jetbrains.com/python/pycharm-professional-2018.3.3.zip",
```
Comment (Ash258) [inline bucket/pycharm-pro.json]: No description

**Comment by Ash258** [comment]
For extras bucket it's recommended to use Toolbox App rather then add new manifests, but it's mainly @r15ch13 decision.

If I could recommend you other solution, I have all JetBrains IDEs with All versions inside [JetBrains bucket](https://github.com/Ash258/Scoop-JetBrains)

**Comment by edgardmessias** [comment]
I make a copy of `pycharm.json`

I will use your bucket

**Pull request closed without merging**